packages/library-base/proguard-rules-consumer-common.pro

```diff
@@ -90,6 +90,9 @@
 -keep class io.realm.kotlin.internal.interop.AsyncOpenCallback {
     *;
 }
+-keep class io.realm.kotlin.internal.interop.NativePointer {
```

Comment:
We should also add these I think? 🤔 

https://github.com/realm/realm-kotlin/issues/1107